src/libsync/capabilities.h

```diff
@@ -48,6 +54,12 @@ class OWNCLOUDSYNC_EXPORT Capabilities
     bool shareResharing() const;
     bool chunkingNg() const;
 
+    /// Returns which kind of push notfications are available
+    PushNotificationTypes pushNotificationsAvailable() const;
+
+    /// Websocket url for files push notifications if available
+    QUrl pushNotificationWebSocketUrl() const;
```

Comment:
I think the url is tightly coupled with the websocket implementation because if we change from websocket to a other protocol we need to do a lot of changes?